cvat/apps/engine/static/engine/js/logger.js

```diff
@@ -152,7 +152,9 @@ class LoggerHandler {
         this._userActivityHandler.updateTimer();
     }
 
-    _closeCallback = event => { this._pushEvent(event); };
+    _closeCallback(event) {
```

Comment:
Is the syntax correct? @bsekachev , could you please look?

Reply:
The syntax is correct, but semantic is not. Actually, we lose the "LoggerHandler" context in this logic when we assign this method to some callback in the row 74 without binding the context.
I would recommend merge develop into this branch.  It would resolve the issue.

Reply:
I changed it. because chrome report error with the original arrow expression.